servers/rendering/renderer_rd/shaders/forward_mobile/scene_forward_mobile.glsl

```diff
@@ -1830,4 +1992,12 @@ void main() {
 #endif //MODE_MULTIPLE_RENDER_TARGETS
 
 #endif //MODE_RENDER_DEPTH
+
+#ifdef MODE_RENDER_MOTION_VECTORS
+	vec3 ndc = screen_position.xyz / screen_position.w;
+	ndc.y = -ndc.y;
+	vec3 prev_ndc = prev_screen_position.xyz / prev_screen_position.w;
```

Comment:
In the Forward+ renderer we multiply these values by 0.5 before doing the subtraction, why are we not doing the same here?